Base DEX Aggregator App

0xsergen opened the pull request:
An app that lets users connect their wallet and perform swaps at the best rates thanks to OpenOcean's API. Uses [Base Bundle](https://marketplace.quicknode.com/bundles/basebundle) add-ons.

**Related [guide](https://github.com/quiknode-labs/docs/pull/1968)**

ferhatqn reviewed (commented):
nice guide overall! some things i noticed

1. wallet balance rounds down to 0 for some reason

<img width="271" alt="image" src="https://github.com/user-attachments/assets/4163bfbf-fb8e-432e-83e5-c81b1569cca7" />

2. rejecting a txn outputs the log larger than the window

<img width="867" alt="Screenshot 2025-05-09 at 11 34 49 AM" src="https://github.com/user-attachments/assets/9ad937cb-d98e-4cea-b984-b3e9ad4eef93" />

3. small swaps dont work without adding 0. as a prefix (e.g., 0.0001)

<img width="551" alt="Screenshot 2025-05-09 at 10 26 32 AM" src="https://github.com/user-attachments/assets/9aa8945d-6121-40bf-aa4f-2880a05c08b0" />

4. wallet history not loading

<img width="589" alt="image" src="https://github.com/user-attachments/assets/6612e6b5-86b6-41f5-ac86-5b26f7980bbb" />

0xsergen commented:
> 1. wallet balance rounds down to 0 for some reason
> 
It's directly from the React library, [Connectkit](https://docs.family.co/connectkit), and not customizable.  We can choose either not to display the balance at the component or leave it as is.

> 2. rejecting a txn outputs the log larger than the window
> 
Updated. Now, error details are logged to the console, but are not displayed on the UI.

> 3. small swaps dont work without adding 0. as a prefix (e.g., 0.0001)
> 
Fixed.

> 4. wallet history not loading
> 
Contacted the marketplace team.
